src/genezio.ts

```diff
@@ -47,24 +46,37 @@ if (ENABLE_DEBUG_LOGS_BY_DEFAULT) {
   setDebuggingLoggerLogLevel("debug");
 }
 
-// program setup
+// super-genezio command
+// commander is displaying help by default for calling `genezio` without a subcommand
+// this is a workaround to avoid that
+// Note: no options can be added to this command
+if (process.argv.length === 2) {
+  GenezioTelemetry.sendEvent({eventType: TelemetryEventTypes.GENEZIO_COMMAND});
+
+  await genezioCommand().catch((error: Error) => {
+    log.error(error.message);
+    GenezioTelemetry.sendEvent({eventType: TelemetryEventTypes.GENEZIO_COMMAND_ERROR, errorTrace: error.message});
+    exit(1);
+  });
+}
+
+// make genezio --version
+program.version(await latestVersion("genezio"), "-v, --version", "Output the current version of genezio.");
```

Comment:
I think this is not the version of genezio installed on the user's machine. This is the latest from `npm`.

You can use `version` from our `utils.ts`.